src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -19,7 +19,7 @@
             + "Contact details can be marked private by prepending 'p' to the prefix.\n"
             + "Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...\n"
             + "Example: " + COMMAND_WORD
-            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney";
+            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25, 119234 t/friends t/owesMoney";
```

Comment:
Hi Yao Cong, this commit message is a little misleading, do take note of that in the future. Also do note to seperate your commits by significant changes or split commits if they are not exactly similar even though they might be related. It would be advisable to commit e.g. in your case, "Implementation/Modifying Address...." and seperate your test update to the subsequent commit.